### PR TITLE
Adds capture logic

### PR DESCRIPTION
**Pull request by hereismyusername**
Adds 'move_to' method and tests

### CODE REVIEW

**Comment by hereismyusername** [comment]
Good point! I've changed those two tests.

**Comment by cldambrosio** [comment]
Hey Selma, I had a look at the changes, I would not have changed the `expect()` lines actually. They are the assumptions which need to be checked once the `move_to!` method is applied. So you'd have:
```
it "captures a piece" do
  @current_piece.move_to!(5, 5)
  @next_square.reload
  expect(@next_square[:is_captured]).to be true
end
it "updates the coordinates of the piece that did the capturing" do
  @current_piece.move_to!(5, 5)
  expect(@current_piece.file).to eq 5
  expect(@current_piece.rank).to eq 5
end
```

File: app/models/piece.rb [hunk 1]
```diff
@@ -49,4 +49,15 @@ def is_obstructed?(col, row)     ## pass in rank and file of the square we want
     end
     false
   end
+
+  def move_to!(new_col, new_row)
+    current_col = self.file
+    current_row = self.rank
+    pieces = Piece.where(file: new_col, rank: new_row, game: game, is_captured: false)
+      captured_piece = pieces.first
```
Comment (cldambrosio) [inline app/models/piece.rb]: quick style change: lines 57 to 61 shouldn't be indented

Comment (hereismyusername) [reply app/models/piece.rb]: Thanks - I've changed that.

**Pull request merged**